Automattic/jetpack
# VideoPress: Consolidate VideoPreview type

While working on a native version of the `Player` component I noticed the `PlayerProps` was a different type definition as the one returned from`usePreview` hook. 
This PR consolidates the preview type in the player props to use the `VideoPreview` type

## Proposed changes:

* Update `PlayerProps.preview` to use the `VideoPreview` type 
* Remove unused properties that were defined in `PlayerProps.preview`

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to block editor
* Confirm that a VideoPress block mounts correctly
